app/views/podcast_planner/_form_main.html.erb

```diff
@@ -0,0 +1,19 @@
+<%= form_with(
```

Comment:
so using Turbo, the function of this controller is just to update the preview, since on submit all it does is `get` back to `index`. this is the "main" middle section of the view, and the target is the "side" section with the preview.

Reply:
Nice!  Seems to work well.